src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -1,13 +1,16 @@
 import React from "react"
 import { View } from "react-native"
+import { ThemeProvider } from "styled-components"
 import styled from "styled-components/native"
 
-import { Fonts } from "../../../data/fonts"
-import Serif from "../../Text/Serif"
 import { Button } from "../Components/Button"
 import { Container } from "../Components/Container"
+import { Divider } from "../Components/Divider"
 import { Timer } from "../Components/Timer"
 import { Title } from "../Components/Title"
+import { Flex } from "../Elements/Flex"
+import Theme from "../Elements/Theme"
```

Comment:
System-wise any reason this is a default export?

Reply:
good catch👍 I'll update the export.